code/headers/uArm_swift_pro_c.hpp

```diff
@@ -1,23 +1,29 @@
-//
-// Created by Slimmerik on 13/04/2019.
-//
+/**
+ * Class uArm_swift_pro provides a implementation for uArm swift pro.
+ */
+
```

Comment:
this part of doxygen should be just above the class, not at the top of the document